setup.py

```diff
@@ -26,15 +24,12 @@ def readfile(name):
 CHANGES = readfile("CHANGES.txt")
 VERSION = '3.0.0.dev0'
 
-PY37MIN = sys.version_info[0] == 3 and sys.version_info[1] >= 7
-
 requires = [
     "Babel",
     "deform >= 2.0.15.dev0",  # .dev0 allows pre-releases.
     "pyramid >= 1.5a1",  # route_name argument to resource_url
```

Comment:
By virtue of our Python support, this should be:

```suggestion
    "pyramid >= 2.0a0",  # Drop Python 3.5 support
```

See https://docs.pylonsproject.org/projects/pyramid/en/main/changes.html#id3